.travis.yml

```diff
@@ -69,6 +69,11 @@ jobs:
       script: .ci/trigger-travis.sh $TRAVIS_ACCESS_TOKEN
       env:
         - TEST_SUITE=none
+    - stage: nightly deploy
+      if: TRAVIS_EVENT_TYPE = cron
```

Comment:
Assuming both this PR and the CI refactor are merged, does this mean that the cron trigger would trigger the other job stages as well (i.e. the unit, pdk, integration tests etc...)? Unless we include a similar `if: TRAVIS_EVENT_TYPE != cron`? 

Also, would `==` not be more readable?

Reply:
Yes it would trigger all the build stages which I think we want since we don't want nightly builds if the code is failing any tests

looks like the travis-ci conditional library supports both `=` and `==` I'll change that line